### PR TITLE
[Streaming] Union api

### PR DESCRIPTION
**Pull request by chaokunyang**
## Why are these changes needed?
Add union java/python api for streaming

## Related issue number

#6184
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: streaming/src/protobuf/remote_call.proto [hunk 1]
```diff
@@ -14,8 +14,8 @@ message ExecutionGraph {
     int32 parallelism = 2;
     NodeType node_type = 3;
     Language language = 4;
-    // serialized user function
-    bytes function = 5;
+    // serialized operator
+    bytes operator = 5;
```
Comment (BalaBalaYi) [inline streaming/src/protobuf/remote_call.proto:18]: no need to build function in python worker?

File: streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/demo/HybridStreamTest.java [hunk 2]
```diff
@@ -34,9 +40,12 @@ public boolean filter(Object value) throws Exception {
     }
   }
 
-  @Test
-  public void testHybridDataStream() throws InterruptedException {
+  @Test(timeOut = 60000)
+  public void testHybridDataStream() throws Exception {
     Ray.shutdown();
```
Comment (BalaBalaYi) [inline streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/demo/HybridStreamTest.java:45]: use @beforemthod for ```Ray.shutdown()```

File: streaming/src/protobuf/remote_call.proto (hunk 1 above)
Comment (chaokunyang) [reply streaming/src/protobuf/remote_call.proto:18]: Put function build in `operator.load_operator`

File: streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/demo/HybridStreamTest.java (hunk 2 above)
Comment (chaokunyang) [reply streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/demo/HybridStreamTest.java:45]: `HybridStreamTest` has only one test, it's not necessary to add `@beforemthod`. If we have multiple tests, we do need to add `@beforemthod`

**Review by BalaBalaYi — APPROVED** [review]

File: streaming/java/streaming-api/src/main/java/io/ray/streaming/api/context/ClusterStarter.java [hunk 3]
```diff
@@ -27,6 +27,7 @@ static synchronized void startCluster(boolean isCrossLanguage, boolean isLocal)
     if (!isLocal) {
       System.setProperty("ray.raylet.config.num_workers_per_process_java", "1");
       System.setProperty("ray.run-mode", "CLUSTER");
+      System.setProperty("ray.redirect-output", "true");
```
Comment (raulchen) [inline streaming/java/streaming-api/src/main/java/io/ray/streaming/api/context/ClusterStarter.java:30]: This option is already removed. Now it's always true.

**Review by raulchen — APPROVED** [review]

**Review by MissiontoMars — APPROVED** [review]

**Pull request merged**